settings/common.py

```diff
@@ -110,8 +110,8 @@
             'email',  
         },
         'APP': {
-            'client_id': 'enter your google client_id',
-            'secret': 'enter your google secret key',
+            'client_id': '507481204731-gpt96umophan45v0q8uk6pu6gs0t9u9f.apps.googleusercontent.com',
```

Comment:
Shouldn't be using this secret in repo IMO. Let's convert this to environment variable as well

Reply:
Yes, but this is an invalid `client_id`.

Reply:
I would prefer to have all secrets as env vars, so that when we run in production we just have to run the container with env vars.

Reply:
Yes, I'll fix this.